### PR TITLE
TTVA-200 | Allow new reservation time to overlap with original time

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Allow updated reservation times to overlap with the original times also in the case that disallow_overlapping_reservations is set to True in the resource unit of the reservation

### CODE REVIEW

File: users/migrations/0012_allow_duplicate_account_emailaddresses.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from django.db import migrations
```
Comment (jopesy) [inline users/migrations/0012_allow_duplicate_account_emailaddresses.py]: This shouldn't be included in this PR.

**Comment by jopesy** [comment]
Just remove the migration from the PR. Otherwise this looks very good.

File: users/migrations/0012_allow_duplicate_account_emailaddresses.py (hunk 1 above)
Comment (jorilindell) [reply users/migrations/0012_allow_duplicate_account_emailaddresses.py]: I already did this change yesterday but forgot to push changes here. Anyway, migration file is now removed
